accounts/abi/bind/bind.go

```diff
@@ -392,10 +392,8 @@ func capitalise(input string) string {
 
 // decapitalise makes a camel-case string which starts with a lower case character.
 func decapitalise(input string) string {
-	// NOTE: This is the current behavior, it doesn't match the comment
-	// above and needs to be investigated.
-	return abi.ToCamelCase(input)
-
+	goForm := abi.ToCamelCase(input)
+	return goForm[:1] + goForm[1:]
```

Comment:
I don't understand ... `goForm[:1] + goForm[1:]` isn't that the same as `goForm` ? Meaning that the change in this file is really just a no-op?

Reply:
oops, yes that's incorrect. Just fixed it. Good catch.